src/main/java/com/box/sdk/BoxLegalHoldPolicy.java

```diff
@@ -330,6 +353,11 @@ public void updateInfo(BoxLegalHoldPolicy.Info info) {
          */
         private String releaseNotes;
 
+        /**
+         * @see #getIsOngoing()
+         */
+        private Boolean isOngoing;
```

Comment:
We usually use the primitive type for these things; how does it work using the class?